unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -322,7 +322,7 @@ notifyNumbered o = case o of
           "",
           endangeredDependentsTable ppeDecl endangerments,
           "",
-          P.wrap "If you want to proceed anyways and leave those definitions without names, use"
+          P.wrap "If you want to proceed anyways and leave those definitions without names, use "
             <> IP.patternName IP.deleteNamespaceForce
```

Comment:
```suggestion
          P.wrap ("If you want to proceed anyways and leave those definitions without names, use"
            <> IP.patternName IP.deleteNamespaceForce)
```
This might work, but I'm not sure.